flask_resources/views/collections.py

```diff
@@ -60,52 +51,49 @@ class ItemView(BaseView):
     def __init__(self, *args, **kwargs):
         """Constructor."""
         super(ItemView, self).__init__(*args, **kwargs)
-        self.item_parser = self.resource.config.item_request_parser
-        self.response_handlers = self.resource.config.item_response_handlers
-        self.request_loaders = self.resource.config.item_request_loaders
+        self._response_handlers = self.resource.config.item_response_handlers
+        self._request_loaders = self.resource.config.request_loaders
 
     def get(self, *args, **kwargs):
         """Get."""
-        _response_handler = self.response_handlers[resource_requestctx.accept_mimetype]
-
         try:
-            return _response_handler.make_response(*self.resource.read(*args, **kwargs))
+            resource_requestctx.request_loader.load_item_request(data=False)
```

Comment:
Shelve: from discussion we might want to `split loader` and `resource_requestctx`:

```
resource_requestctx.update(request_loader.load_item_request(data=False))
```

Clearer when we use it...

Reply:
yup: https://github.com/inveniosoftware/flask-resources/issues/38